packages/metadataform/src/form.tsx

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+/**
+ * @packageDocumentation
+ * @module metadataform
+ */
+
+import React from 'react';
+import Form, { IChangeEvent } from '@rjsf/core';
+import { JSONSchema7 } from 'json-schema';
+import { PartialJSONValue, ReadonlyPartialJSONObject } from '@lumino/coreutils';
+import { ReactWidget } from '@jupyterlab/apputils';
+import { RJSFTemplatesFactory } from '@jupyterlab/ui-components';
+
+import { MetadataForm } from './token';
+
+/**
+ * A ReactWidget with the form itself.
+ */
+export class FormWidget extends ReactWidget {
+  /**
+   * Constructs a new FormWidget.
+   */
+  constructor(props: MetadataForm.IProps, pluginId?: string) {
+    super();
+    this.addClass('jp-ReactWidget');
```

Comment:
I would change the name of the class because it is misleading on the component it is applied to.

```suggestion
    this.addClass('jp-FormWidget');
```